zh/_syntax.md

```diff
@@ -347,7 +311,9 @@ Sass 中一个正在被众多开发者滥用的功能，就是**选择器嵌套*
 
 选择器越具体则声明语句越冗长，而且对最近选择器的引用(`&`)也越频繁。在某些时候，出现混淆选择器路径和探索下一级选择器的错误率很高，这非常不值得。
 
-为了防止此类情况，我们**应该尽可能避免选择器嵌套**。然而，显然只有少数情况适应这一措施。
+为了防止此类情况，我们今年就 [the Inception rule](http://thesassway.com/beginner/the-inception-rule) 讨论了很多很多。它建议嵌套不要超过三层，我的一件比较激进，**建议尽量避免使用嵌套**。 
+
+虽然我们在下一节看到这条规则有一些例外，但这一观点还是很受欢迎的。更多信息请阅读：[Beware of Selector Nesting](http://www.sitepoint.com/beware-selector-nesting-sass/) 和 [Avoid nested selectors for more modular CSS](http://thesassway.com/intermediate/avoid-nested-selectors-for-more-modular-css).
```

Comment:
No appropriate chinese words to describe "the Inception rule" here.

Other labels will be translated.